scripts/changelog_check.rb

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env ruby
 require 'open3'
 require 'optparse'
+require 'active_support/inflector'
```

Comment:
This will likely require some other changes to run the changelog script in CI in a different stage.  Currently, it runs alongside the "install" stage since it doesn't have any bundled dependencies

Reply:
> This will likely require some other changes to run the changelog script in CI in a different stage. Currently, it runs alongside the "install" stage since it doesn't have any bundled dependencies

Maybe we should just remove the formatting, then?

https://github.com/18F/identity-idp/pull/6880#discussion_r959561620

Reply:
I'm good with that!

Reply:
Removed in 316b3c00c.